quil-rs/src/instruction/mod.rs

```diff
@@ -413,7 +413,7 @@ pub fn get_expression_parameter_string(parameters: &[Expression]) -> String {
         return String::from("");
     }
 
-    let parameter_str: String = parameters.iter().map(|e| format!("{e}")).collect();
+    let parameter_str: String = parameters.iter().map(|e| format!(r"{e}")).collect();
```

Comment:
What's the purpose of the raw string here?

Reply:
Oops, good catch, I was testing something else that needed it and forgot about it. Cleaned it up.